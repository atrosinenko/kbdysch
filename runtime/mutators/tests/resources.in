[ENV:KBDYSCH_MUTATOR_NUM_SPLICES=0]
[BEGIN:DUMPING OUTPUTS]
[END:CALL DEINIT]
[MUTATE]
%Section 1 %N00 %N01
%Section 2 %R01
%Section 3
%Section 4 %N02
%Section 5 %N03 %R02
%Section 6 %R03
%Section 7 %R00 %R03
