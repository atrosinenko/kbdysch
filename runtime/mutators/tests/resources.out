INJECTOR: Mutator requested 7 mutations.
INJECTOR: OUTPUT #0: BEGIN
INJECTOR: %Section 3
INJECTOR: %Section 4 %N02
INJECTOR: %Section 5 %N03 %R00
INJECTOR: %Section 6 %R01
INJECTOR: OUTPUT #0: END
INJECTOR: OUTPUT #1: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 3
INJECTOR: %Section 4 %N02
INJECTOR: %Section 5 %N03 %R02
INJECTOR: %Section 6 %R03
INJECTOR: %Section 7 %R00 %R03
INJECTOR: OUTPUT #1: END
INJECTOR: OUTPUT #2: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 2 %R01
INJECTOR: %Section 4 %N02
INJECTOR: %Section 5 %N03 %R02
INJECTOR: %Section 6 %R03
INJECTOR: %Section 7 %R00 %R03
INJECTOR: OUTPUT #2: END
INJECTOR: OUTPUT #3: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 2 %R01
INJECTOR: %Section 3
INJECTOR: OUTPUT #3: END
INJECTOR: OUTPUT #4: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 2 %R01
INJECTOR: %Section 3
INJECTOR: %Section 4 %N02
INJECTOR: OUTPUT #4: END
INJECTOR: OUTPUT #5: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 2 %R01
INJECTOR: %Section 3
INJECTOR: %Section 4 %N02
INJECTOR: %Section 5 %N03 %R02
INJECTOR: %Section 7 %R00 %R03
INJECTOR: OUTPUT #5: END
INJECTOR: OUTPUT #6: BEGIN
INJECTOR: %Section 1 %N00 %N01
INJECTOR: %Section 2 %R01
INJECTOR: %Section 3
INJECTOR: %Section 4 %N02
INJECTOR: %Section 5 %N03 %R02
INJECTOR: %Section 6 %R03
INJECTOR: OUTPUT #6: END
